src/test/osd/RadosModel.h

```diff
@@ -2729,10 +2909,14 @@ class TierFlushOp : public TestOp {
     cout << num << ":  got " << cpp_strerror(r) << std::endl;
     if (r == 0) {
       // sucess
+    } else if (r == -EBUSY) {
+      // could fail if snap is not oldest
+    } else if (r == -ENOENT) {
+      // could fail if obj is removed
```

Comment:
Or this?

Reply:
I added a specific condition to check ENOENT.